pkg/wasm/cache.go

```diff
@@ -61,7 +62,7 @@ type LocalFileCache struct {
 	// Map from Wasm module checksum to cache entry.
 	modules map[moduleKey]*cacheEntry
 	// Map from tagged URL to checksum
-	checksums map[string]string
+	checksums map[string]*checksumEntry
```

Comment:
nit: do we need it to be a pointer? Seems to mostly introduce possibility for errors; copying the checksumEntry struct should be low cost

Reply:
Will consider later. Thanks!